src/auth/refreshclient.ts

```diff
@@ -107,8 +109,10 @@ export class UserRefreshClient extends OAuth2Client {
         refresh_token: this._refreshToken,
         target_audience: targetAudience,
       } as {}),
-    });
+    };
+    AuthClient.setMethodName(opts, 'fetchIdToken');
```

Comment:
Completely optional: Since these classes extend `AuthClient` we could use the class's `setMethodName` and avoid importing `AuthClient`.

```suggestion
    UserRefreshClient.setMethodName(opts, 'fetchIdToken');
```

Here and in other places

Reply:
Actually, after thinking about it, I'm not a super fan of static inheritance like that. I think it makes it more obscure what's actually being called. (Also, several of the other usages were not deriving from anything that had `setMethodName` so the usage was mixed.) I'm okay with it either way, but maybe leaning toward not changing it.